app/api/ltc.py

```diff
@@ -153,4 +152,33 @@ def aggregate_outbreaks():
     flask.current_app.logger.info('Collapsing %s data took %.1f seconds, %d to %d rows' % (
         state, (t2 - t1), df.shape[0], processed_df.shape[0]))
 
+    return processed_df
+
+
+@api.route('/aggregate-outbreaks', methods=['POST'])
+def api_aggregate_outbreaks():
+    payload = flask.request.data.decode('utf-8')
+    df = pd.read_csv(io.StringIO(payload))
+    processed_df = do_aggregate_outbreaks(df)
+
     return Response(processed_df.to_csv(index=False), mimetype='text/csv')
+
+
+def cli_aggregate_outbreaks(url):
+    # extract the parameters from the google docs url and formulate a CSV export url
+    m = re.search('.*\/d\/(.*)\/edit.*#gid=(.*)', url)
+    if m:
+        key = m.group(1)
+        gid = m.group(2)
+        url = f"https://docs.google.com/spreadsheets/d/{key}/export?format=csv&gid={gid}"
+    else:
+        click.echo('Invalid Google Sheets URL')
+        raise click.Abort()
+
+    # fetch the CSV data
+    with urllib.request.urlopen(url) as response:
+        data = response.read()
+
+    # process it and print the result to STDOUT
+    processed_df = do_aggregate_outbreaks(pd.read_csv(io.StringIO(data.decode('utf-8'))))
+    print(processed_df.to_csv(index=False))
```

Comment:
maybe this should take an output file arg instead of printing to stdout? not sure how this works with all the logging statements and separating the two